docs/modules/operation/pages/database-reports/templates/standard.adoc

```diff
@@ -1,8 +1,9 @@
 
 //This file includes standard text for the format and delivery sections of each report template dialog.
 
-In addition, the report screen includes delivery and schedule options.
-Click the *Deliver this report* (for report delivery options) and *Schedule this report* (for the schedule editor) checkboxes to specify this information.
+The last two parameters in this report are checkboxes for delivery and scheduling of the report.
+Checking the "Deliver this report" checkbox will provide the delivery options described below.
+Checking the "Schedule this report" checkbox will open up the scheduling editor, which is described in the Schedule Editor section below.
 
 === Report Delivery Options
```

Comment:
Line 22/23: Should be an extra blank line between "Specify the following types of schedule" and  "* Daily" to start a new list.  Should also be something closer to "Specify the desired schedule to automate report delivery"